src/main.ts

```diff
@@ -21,6 +22,12 @@ v1.route("/event")
 v1.route("/checkin")
   .post(create_check_in)
 
+v1.route("/client")
```

Comment:
Yeah there's a lot of argument about this... I think to be consistent with what we have done in other routes we should keep it singular. In RESTful applications, whenever you ping `GET /client` rather than `GET /client/:id` you expect to get all, while when you do `:id` you expect to get one with the given ID.

Reply:
LMK what you think about that. I could go either way- just know it would mean rework on some of our existing codebases.